### PR TITLE
Feature/netfoley npc dodge

### PR DESCRIPTION
**Pull request by NetFoley**
c + mieu

### CODE REVIEW

File: Scenes/Characters/NPC/Fighting.gd [hunk 1]
```diff
@@ -3,7 +3,7 @@ func is_class(value: String): return value == "" or .is_class(value)
 func get_class() -> String: return ""
```
Comment (Weact) [inline Scenes/Characters/NPC/Fighting.gd:3]: Please add a classname : I.E. NpcFightingState and fill the is_class and get_class methods

File: Scenes/Characters/NPC/Fighting.gd [hunk 2]
```diff
@@ -3,7 +3,7 @@ func is_class(value: String): return value == "" or .is_class(value)
...
 
 var offAngle : float = PI
-var min_dist = 0.0
+var min_dist = 32.0
 var max_dist = 128.0
```
Comment (Weact) [inline Scenes/Characters/NPC/Fighting.gd:7]: These two should be typed with ` : type `
i.e. var min_dist : float = 32.0

File: Scenes/Characters/NPC/Fighting.gd [hunk 3]
```diff
@@ -3,7 +3,7 @@ func is_class(value: String): return value == "" or .is_class(value)
 func get_class() -> String: return ""
 
 var offAngle : float = PI
```
Comment (Weact) [inline Scenes/Characters/NPC/Fighting.gd:5]: why do you declare a PI veriable if PI is already a global variable?

File: Scenes/Characters/NPC/Distancing.gd [hunk 4]
```diff
@@ -6,8 +6,11 @@ func get_class() -> String: return ""
```
Comment (Weact) [inline Scenes/Characters/NPC/Distancing.gd:6]: Same here, should add a class_name and fill is_class and get_class methods

**Review by Weact — CHANGES_REQUESTED** [review]
Add class names to Kitting.gd, Fighting.gd, Distancing.gd
Type variables

File: Scenes/Characters/NPC/Fighting.gd (hunk 3 above)
Comment (NetFoley) [reply Scenes/Characters/NPC/Fighting.gd:5]: I don't declare pi variable. PI is just the default value for this variable

**Review by Weact — APPROVED** [review]

**Pull request merged**